lib/_http_outgoing.js

```diff
@@ -847,15 +836,12 @@ OutgoingMessage.prototype._flushOutput = function _flushOutput(socket) {
     socket.cork();
   }
 
-  const outputLength = this.outputData.length;
-  if (outputLength <= 0)
+  if (this.outputData.length <= 0)
     return undefined;
 
-  const outputData = this.outputData;
   socket.cork();
   let ret;
-  for (var i = 0; i < outputLength; i++) {
-    const { data, encoding, callback } = outputData[i];
```

Comment:
We should not make this change for performance reasons.

Reply:
resolved in https://github.com/nodejs/node/pull/30958/commits/17049d56621869d8776a922d5e56b6c5ec35be7b